hapi/error.py

```diff
@@ -55,10 +62,6 @@ def __init__(self, result, request, err=None):
         self.err = err
 
     def __str__(self):
-        return self.__unicode__().encode('ascii', 'replace')
-
-
-    def __unicode__(self):
```

Comment:
There is no builtin ```__unicode__``` function in python3. ```@python_2_unicode_compatible``` is for python2 compatibility. It has no effect in python3.
But it will encode unicode string with ```utf8``` as returned value of function ```__str__``` in python2, which has the same behavior as before.